src/screens/DomainScreen/elements/Header/elements/CredderInfoGroup.js

```diff
@@ -73,14 +73,14 @@ const styles = StyleSheet.create({
     },
     tooltipShadowContainer: {
         paddingHorizontal: 10,
-        shadowColor: '#000',
+        shadowColor: COLORS.black,
         shadowOffset: {
             width: 0,
             height: 12,
         },
         shadowOpacity: 0.58,
         shadowRadius: 16,
-        shadowColor: 'black',
+        shadowColor: COLORS.black,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There is a redundancy in the code. The `shadowColor` property has been declared twice with the same value `COLORS.black`. This could lead to confusion and potential bugs in the future. You should remove one of them.
```diff
     tooltipShadowContainer: {
         paddingHorizontal: 10,
-        shadowColor: COLORS.black,
         shadowOffset: {
             width: 0,
             height: 12,
         },
         shadowOpacity: 0.58,
         shadowRadius: 16,
         shadowColor: COLORS.black,
         elevation: 16,
     }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->